lms/djangoapps/certificates/tests/factories.py

```diff
@@ -24,6 +24,7 @@ class Meta(object):
     mode = GeneratedCertificate.MODES.honor
     name = ''
     verify_uuid = uuid4().hex
+    grade = ''
```

Comment:
Why we need `grade` variable here?

Reply:
I just wanted to make sure that grade is also overridden.